source/guide_shiori.rst

```diff
@@ -0,0 +1,112 @@
+.. highlight:: console
+
+.. author:: Kevin Jost <https://github.com/systemsemaphore>
+
+.. tag:: lang-go
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/shiori.svg
+      :align: center
+
+######
+Shiori
+######
+
+.. tag_list::
+
+Shiori_ is a simple bookmarks manager written in Go and distributed under the MIT License. It can be used as a `command line application <https://github.com/go-shiori/shiori/wiki/Usage#using-command-line-interface>`_ or via the built-in web-ui.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`web backends <web-backends>`
+
+Prerequisites
+=============
+
+.. note:: Shiori won't work with a subpath (e.g. https://isabell.uber.space/shiori/).
+
+By default, Shiori runs on port 8080.
+To reach it, you need to configure a `web backend <webbackend_>`_.
+
+::
+
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 8080
+  Set backend for / to port 8080; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+.. _webbackend: https://manual.uberspace.de/web-backends.html
+
+Installation
+============
+
+Like a lot of Go software, Shiori is distributed as a single binary. Download Shiori's latest `release <https://github.com/go-shiori/shiori/releases/latest>`_ and make sure that the file can be executed.
+
+::
+
+  [isabell@stardust ~]$ mkdir ~/shiori && cd ~/shiori
```

Comment:
The app writes all data to `~/.local/share/shiori` and there aren't any configuration files. What do you think about saving the binary as `~/bin/shiori` using `wget ... -O ~/bin/shiori` and skipping the directory step entirely?